### DISCUSSION TITLE
AMF Export?

### DISCUSSION THREAD

**Discussion by Durahl** [supermerill/SuperSlicer#2781] [Q&A]
Greetings!

I'm currently trying to _**recreate**_ a thing where I created a Platter with all the necessary changes to a Mesh ( Mesh Modifiers, Layer Height specific Edits, etc... ) to share with someone _without_ the Export _also_ including things _only_ relevant to _my_ 3D Printer like the Print Bed, Speeds, Print Profile, etc... that could mess with the other persons Printer.

I _know_ I once managed to do this using an [AMF Fileformat](https://en.wikipedia.org/wiki/Additive_manufacturing_file_format) Export because I still have the two files, I created back then in my public OneDrive folder, but I cannot find the Export feature in either SuperSlicer or PrusaSlicer anymore? Was it cut? 🤔

**Comment by supermerill** [comment]
The menu option was commented out by prusa.
`// Deprecating AMF export. Let's wait for user feedback.`

The amf should contains the same information than the 3mf.

You can kind of do what you want manually:
open the 3mf (it's a zip) and remove the metadata/config files.
the model config store the config per object
the _layer_config_ranges.xml contains layer range modifiers

**Reply by Durahl** [reply]
Hmm... So, I wasn't just imagining things 🤔 Any chance to get it commented back in for convenience's sake?

Removing the unnecessary files is certainly not _that_ big of a deal but man does it get old fast 😁
Or... IDK... Have a checkbox ✅ to not include System Specific Settings when creating a 3MF?

I find it actually really weird to include machine specific settings to begin with... What's an Ender 3 supposed to do with my Voron 2.4 settings having it run at 300mm/s @5k Accel when I just want to share an already prepared platter with someone? 🤨

**Reply by supermerill** [reply]
It's the project file, so you can recover all your settings later, if you need to change a value and reslice.

> Any chance to get it commented back in for convenience's sake?

I don't think you need it.

> Have a checkbox ✅ to not include System Specific Settings when creating a 3MF?

so you want to export your platter as 3mf, like the export platter as stl?

**Reply by Durahl** [reply]
> It's the project file, so you can recover all your settings later, if you need to change a value and reslice.

I get the appeal of _that_ concept - But it's not something _**I**_ make use often - To be frank I kinda hate it because by the time I need to reprint something chances are high my preferred settings may have changed and the 3MF will now import back old settings forcing me to switch back to my new settings - Not the end of the world but... Nyeeh 🤨

> so you want to export your platter as 3mf, like the export platter as stl?

No, all I want is to be able to hand people [a file like this](https://1drv.ms/u/s!Ali6LwKc7R0fqI8A-TESFch4KDfROQ?e=UkmiAJ) with a plethora of edits to the meshes I did without them possibly using _either_ my **Print** or **Printer** Profiles **_and_** ideally also without me having to dick around in a ZIP File to remove the problematic content it contains 😁

- A 3MF will overdeliver by default including my potentially dangerous Print and Printer Settings for someone not familiar with it.
- An AMF will deliver just right - The files with my edits while relying on whatever Settings the end-user has set up so far.
- A STL will underdeliver by stripping the plate of all information except the bare mesh I edited into it.

IDK... Still having access to a fileformat that doesn't screw around with other peoples printers should kinda be a nobrainer? 🤔

**Reply by supermerill** [reply]
if I allow to export platter as 3mf, it's enough?
only meshes, like the stl but separated like in the amf/3mf.

amf and 3mf have no real difference. It's just a hassle to maintain both.

**Reply by Durahl** [reply]
Sorry for the late reply...

Are you suggesting of outright stripping away features from 3MF to bring it in line with AMF? That would be terrible in its own way for those relying on those features. I mean... Like I said... If you can add something like a checkbox feature during Exporting to include/exclude **Print** and/or **Printer** Configs, then sure enough - That should be something to work for everyone but if **hard removing** them is the only option, then the obvious choice is to either keep AMF as an alternative option ( enable its flag Prusa disabled? 🤔 ) or tell ppl to suck it up and manually delete the files inside the 3MF 😭

**Reply by supermerill** [reply]
People already give me 3mf without any profile, because it's downloaded from somewhere.
